src/libraries/Common/tests/System/Threading/Tasks/TaskTimeoutExtensions.cs

```diff
@@ -95,28 +94,37 @@ public static async Task WhenAllOrAnyFailed(this Task[] tasks)
         private static Task WhenAllOrAnyFailedCore(this Task[] tasks)
         {
             int remaining = tasks.Length;
-            var tcs = new TaskCompletionSource<bool>();
+            var tcs = new TaskCompletionSource();
```

Comment:
Does this file not get used in any tests that target .NET Framework, e.g. with WinHttpHandler?

Reply:
It probably does.